.eslintrc.json

```diff
@@ -174,9 +174,7 @@
             }
         ],
         "react/jsx-key": "error",
-        "react/jsx-indent": "error",
```

Comment:
as far as i tested it, this rule should not make problem at the moment, right?

would propose the following config:
```
        "react/jsx-indent": [
            "error",
            4,
            {
                "indentLogicalExpressions": true
            }
        ],
```

Reply:
No, that one didn't make any problems, but I didn't find a case where it made a difference, that's why I removed it. But it seems like you did :slightly_smiling_face: Testing what you are suggesting, and will add it afterwards.

Reply:
Just added it, the additional configuration is superb :slightly_smiling_face: